try{
    console.log('start');
    // throw new Error('error message');
    console.log('end');
}catch(e){
    console.log(e);
}finally{
    console.log('bye');
}